concepticondata/conceptlists/Sidwell-2021-100.tsv

```diff
@@ -0,0 +1,101 @@
+Sidwell-2021-100-21	21	Egg	39	24	744	EGG
+Sidwell-2021-100-22	22	Eye	39	25	1248	EYE
+Sidwell-2021-100-23	23	Fire	48	27	221	FIRE
+Sidwell-2021-100-24	24	Flesh/meat	96	28	2615	FLESH OR MEAT
+Sidwell-2021-100-25	25	Fish	49	29	227	FISH
+Sidwell-2021-100-26	26	Fly(v.)		30	1441	FLY (MOVE THROUGH AIR)
+Sidwell-2021-100-27	27	Foot	56	31	1301	FOOT
+Sidwell-2021-100-28	28	Full(container)		32	1429	FULL
+Sidwell-2021-100-29	29	Give	60	33	1447	GIVE
+Sidwell-2021-100-30	30	Good	61	34	1035	GOOD
+Sidwell-2021-100-31	31	Green		36	1425	GREEN
+Sidwell-2021-100-32	32	Hair(of head)	65	37	2648	HAIR (HEAD)
+Sidwell-2021-100-33	33	Hand	65	38	1277	HAND
+Sidwell-2021-100-34	34	Head		39	1256	HEAD
+Sidwell-2021-100-35	35	Hear	69	40	1408	HEAR
+Sidwell-2021-100-36	36	Hot	179	43	1286	HOT
+Sidwell-2021-100-37	37	I	78	44	1209	I
+Sidwell-2021-100-38	38	Kill(v.)	82	45	1417	KILL
+Sidwell-2021-100-39	39	Knee		46	1371	KNEE
+Sidwell-2021-100-40	40	Know	83	47	3626	KNOW
+Sidwell-2021-100-41	41	Leaf		48	628	LEAF
+Sidwell-2021-100-42	42	Long(length)		51	1203	LONG
+Sidwell-2021-100-43	43	Louse		52	1392	LOUSE
+Sidwell-2021-100-44	44	Man/Male/husband		53	1554	MAN
+Sidwell-2021-100-45	45	Woman/Female/wife			962	WOMAN
+Sidwell-2021-100-46	46	Many	95	54	1198	MANY
+Sidwell-2021-100-47	47	Moon	96b	55	1313	MOON
+Sidwell-2021-100-48	48	Mouth		57	674	MOUTH
+Sidwell-2021-100-49	49	Name	100	58	1405	NAME
+Sidwell-2021-100-50	50	Neck	103	59	1333	NECK
+Sidwell-2021-100-51	51	New		60	1231	NEW
+Sidwell-2021-100-52	52	Night	105	61	1233	NIGHT
+Sidwell-2021-100-53	53	Nose	106	62	1221	NOSE
+Sidwell-2021-100-54	54	one	109	64	1493	ONE
+Sidwell-2021-100-55	55	Path/Road	120	65	2457	PATH OR ROAD
+Sidwell-2021-100-56	56	Rain	115	67	2108	RAINING OR RAIN
+Sidwell-2021-100-57	57	Red		68	156	RED
+Sidwell-2021-100-58	58	Root		69	670	ROOT
+Sidwell-2021-100-59	59	Sand	126	71	671	SAND
+Sidwell-2021-100-60	60	Say/Speak		72	1458	SAY
+Sidwell-2021-100-61	61	See	130	73	1409	SEE
+Sidwell-2021-100-62	62	Seed	131	74	714	SEED
+Sidwell-2021-100-63	63	Sit	136	75	1416	SIT
+Sidwell-2021-100-64	64	Skin	137	76	763	SKIN
+Sidwell-2021-100-65	65	Sleep	139	77	1585	SLEEP
+Sidwell-2021-100-66	66	Small	140	78	1246	SMALL
+Sidwell-2021-100-67	67	Smoke(n.)	142	79	778	SMOKE (EXHAUST)
+Sidwell-2021-100-68	68	Stand	151	80	1442	STAND
+Sidwell-2021-100-69	69	Star	152	81	1430	STAR
+Sidwell-2021-100-70	70	Stone	154	82	857	STONE
+Sidwell-2021-100-71	71	Sun	157	83	1343	SUN
+Sidwell-2021-100-72	72	Swim	159	84	1439	SWIM
+Sidwell-2021-100-73	73	Tail	160	85	1220	TAIL
+Sidwell-2021-100-74	74	Tongue		88	1205	TONGUE
+Sidwell-2021-100-75	75	Tooth		89	1380	TOOTH
+Sidwell-2021-100-76	76	Tree		90	906	TREE
+Sidwell-2021-100-77	77	two	176	91	1498	TWO
+Sidwell-2021-100-78	78	Walk	178	92	1443	WALK
+Sidwell-2021-100-79	79	Water(fresh)	181	94	948	WATER
+Sidwell-2021-100-80	80	We(pl.incl.)	182	95	1131	WE (INCLUSIVE)
+Sidwell-2021-100-81	81	White	187	97	1335	WHITE
+Sidwell-2021-100-82	82	You(sg.)	168	100	3292	THOU (HONORIFIC)
```

Comment:
This should be mapped to `1215 THOU` as it is not the honorific form

Reply:
Important, yes!

Reply:
Same here, back to `THOU`